refextract/references/kbs.py

```diff
@@ -42,6 +43,23 @@
 from ..documents.text import re_group_captured_multiple_space
 
 
+@contextlib.contextmanager
+def file_resolving(fpath, reader=None, **kwargs):
+    if isinstance(fpath, six.string_types):
+        print('Load kb path %s' % fpath)
```

Comment:
They are actually not expeceted by any code. They are only useful when using refextract by CLI, for debugging purposes. But then indeed, would be great to look through all refextract and change all the prints for the proper logging.debug/info/warning... commands.